include/openspace/navigation/path.h

```diff
@@ -106,6 +111,12 @@ class Path {
      */
     glm::dquat easedSlerpRotation(double t) const;
 
+    /**
+     * Compute the interpolated rotation quaternion using a method that is
+     * customized to look good for linear paths
```

Comment:
Maybe possible to give some more information about the nature of the customization?   Should have a parameter explanation for the `t` parameter, too